docker/Dockerfile

```diff
@@ -216,6 +228,25 @@ RUN set -e; \
     esac; \
     truncate -s0 ~/.bash_history;
 
+RUN set -e; \
+    if [ -z $(which bazel) ]; then \
+        BAZEL_INSTALL_DIR='/usr/local/bin/'; \
+        BAZEL_VERSION='5.2.0'; \
+        BAZEL_INSTALL_PATH="$BAZEL_INSTALL_DIR/bazel-$BAZEL_VERSION"; \
+        BAZEL_ARCH=$(case "$TARGETARCH" in \
+                     'amd64') \
```

Comment:
Indent normally, not to align with case.
And I would simply say `sed 's/^amd64$/x86_64/ <<< "$XXX"'`

Reply:
I would rather use case for explicity

Reply:
case would be over complicated and if you have many, you'll need to sacrifice something more important for it, e.g. you're merging components in its sorted order to a separated block.

Reply:
There're distro-specific assertion on top for lllvm, you can take them as examples.

Reply:
`case` will show the equilavalent groups more clearly, and that is more scalable when you have more to add.

Reply:
> and that is more scalable when you have more to add.

Not really, because the abstraction won't scale.
But anyway, this is more of an experience thing, I don't have solid argument to say yes or no to this.